components/mailgun/actions/send-email/send-email.js

```diff
@@ -0,0 +1,104 @@
+const mailgun = require("../../mailgun.app.js");
+
+module.exports = {
+  key: "mailgun-send-email",
+  name: "Mailgun Send Email",
+  description: "Send email with Mailgun.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mailgun,
+    domain: {
+      propDefinition: [
+        mailgun,
+        "domain",
+      ],
+    },
+    fromName: {
+      type: "string",
+      label: "From Name",
+      description: "Sender name",
+    },
+    from: {
+      type: "string",
+      label: "From Email",
+      description: "Sender email address",
+    },
+    replyTo: {
+      type: "string",
+      label: "Reply Email",
+      description: "Sender Reply-To email address",
+    },
+    to: {
```

Comment:
Should `cc` and `bcc` be supported as optional string arrays, as well?

Reply:
Good catch!